ansibullbot/triagers/ansible.py

```diff
@@ -552,6 +555,8 @@ def create_actions(self, iw, actions, valid_labels):
                 if 'owner_pr' in iw.labels:
                     actions.unlabel.append('owner_pr')
 
+        """
+        FORK: PRs outside of ansible/ansible are expected to be community_review so we don't need to add these.
```

Comment:
This will likely be removed upstream as well since all PRs in `ansible/ansible` are expected to be `core_review`.